cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -55,7 +55,7 @@
       cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs')
       cronjob at:'30 16,17 * * *', do:deploy_dir('bin', 'cron', 'update_dotd')
       cronjob at:'0 * * * *', do:deploy_dir('bin', 'cron', 'start_broken_link_checker'), notify:'dev+crontab@code.org'
-      cronjob at:'0 19 * * 1-5', do:deploy_dir('bin', 'cron', 'commit_content')
+      cronjob at:'0 20 * * 1-5', do:deploy_dir('bin', 'cron', 'commit_content')
```

Comment:
Is this the right spot?  I don't know how to test besides merge and see what happens....

Reply:
thanks for tackling this Erin! It does look like this is one line that needs to be updated, but I would suggest looking at the last time this line was modified to see if you can find all of the lines that need to be updated. things this should include:
* staging noon commit (this line)
* levelbuilder noon commit
* 12:05pm DTS
* levelbuilder 3:something commit
* 3:something DTL

I am not sure if there are others.

Reply:
I think @sureshc  already covered those in https://github.com/code-dot-org/code-dot-org/pull/31536